### DISCUSSION TITLE
[Feature] Support internal and external URLs for server

### DISCUSSION THREAD

**Discussion by imran0** [immich-app/immich#2290] [Feature Request]
### The feature

I’d really like Immich to have the availability to support an external URL as well as an internal one so that I could have a local IP address to access Immich and a external IP/domain when I’m outside of my house (I.e. when using services like Tailscale).

Apps like Home Assistant support this

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #308
